### PR TITLE
Spelling, grammar, and layout fixes

### PR DESCRIPTION
**Pull request by alexyorke**
This PR contains a collection of grammar, spelling, and layout fixes, including making phrasing more consistent, fixing broken links, and clarifying and fixing incorrect instructions.

The scope of these changes are for images, urls, menu, and more. Also, it includes some things which were not completely finished in other issues (e.g. `code styling` for paths.)

These also include installation instructions for `macOS`.

This is related to issue 2034 internally.

### CODE REVIEW

**Review by JohnnyGitH — APPROVED** [review]

File: _pages/docs/request-match-rules/mockdefinition.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: single
-title: Mockdefinition
+title: mockdefinition
```
Comment (mariareyesfoci) [inline _pages/docs/request-match-rules/mockdefinition.md]: for the title can we keep the capital M. for title it makes sense that the first work is capitalized

File: _pages/docs/request-match-rules/mockdefinition.md [hunk 2]
```diff
@@ -9,29 +9,29 @@ classes: wide
...
 
-The OpenApi file defines a standard, programming language-agnostic interface description for APIs, allowing humans and computers to understand the capabilities of a service without source code.
+The OpenAPI file defines a standard, programming language-agnostic interface description for APIs, allowing humans and computers to understand the capabilities of a service without source code.
 
-The Mockdefinition is generated by the Orbital Designer. This allows the user to add, update or delete Scenarios.
+The mockdefinition is generated by the designer. This allows the user to add, update or delete Scenarios.
 
 ### Scenarios
 
 The Scenarios describe what the endpoint will expect from the user's HTTP request and the response the user
```
Comment (mariareyesfoci) [inline _pages/docs/request-match-rules/mockdefinition.md]: ```The Scenarios describe``` to ```the scenarios describe```

File: _pages/docs/request-match-rules/mockdefinition.md [hunk 3]
```diff
@@ -9,29 +9,29 @@ classes: wide
...
 should expect from the endpoint. A scenario consists of:
 
-- **Metadata:** A title and description to explain the purpose of the scenario.
-- **Request Match Rules:** These rules will indicate what the users HTTP request has to match to get a response from the server.
-- **Response:** The server's response to the user's HTTP request.
+- **Metadata:** a title and description to explain the purpose of the scenario.
+- **request match rules:** these rules will indicate what the users HTTP request has to match to get a response from the server.
```
Comment (mariareyesfoci) [inline _pages/docs/request-match-rules/mockdefinition.md]: can we capitalize ```Request```?

Comment (alexyorke) [reply _pages/docs/request-match-rules/mockdefinition.md]: Good find; fixed

File: _pages/docs/request-match-rules/mockdefinition.md (hunk 2 above)
Comment (alexyorke) [reply _pages/docs/request-match-rules/mockdefinition.md]: Good find; fixed

File: _pages/docs/request-match-rules/mockdefinition.md (hunk 1 above)
Comment (alexyorke) [reply _pages/docs/request-match-rules/mockdefinition.md]: Good find; fixed

**Review by mariareyesfoci — APPROVED** [review]

**Pull request merged**